### Same "cl:functions:echo-stream-output-stream.diff"
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:functions:echo-stream-input-stream.txt	2017-04-27 21:43:51.000000000 +0100
+++ saved-pages-clus/cl:functions:echo-stream-input-stream.txt	2017-04-27 21:43:51.000000000 +0100
@@ -3,1 +4,1 @@
 Syntax


-echo-stream-input-stream echo-stream => input-stream
+ echo-stream-input-stream  echo-stream → input-stream
+ echo-stream-output-stream echo-stream → output-stream

-echo-stream-output-stream echo-stream => output-stream

 Arguments and Values

@@ -12,18 +16,51 @@
 echo-stream - an echo stream.
 input-stream - an input stream.
 output-stream - an output stream.
 Description

 echo-stream-input-stream returns the input stream from which echo-stream receives input.
 echo-stream-output-stream returns the output stream to which echo-stream sends output.
-Examples None.

-Side Effects None.

-Affected By None.
+Examples
+None.
+Side Effects
+None.
+Affected By
+None.
+Exceptional Situations
+None.
+See Also
+None.
+Notes

-Exceptional Situations None.
+None.

-See Also None.

-Notes None.
\ No newline at end of file
+\
\ No newline at end of file
